server/src/graphql/resolvers/mutations.js

```diff
@@ -134,8 +139,27 @@ module.exports = {
       model.id = uuidv4();
     }
     try {
-      await db.Well.upsert({ ...model });
-      return db.Well.findByPk(model.id);
+      if (
+        model.status === 'active' ||
```

Comment:
Is there ever a point in which this if statement doesn't evaluate as true? I feel like you can skip this if and go straight to your second if from line 147

Reply:
just put that as an extra layer of checking but you're right, it would never make it to this point and still evaluate at false